helm/fdb-operator/values.yaml

```diff
@@ -1,12 +1,71 @@
-operator:
-  name: fdb-kubernetes-operator-controller-manager
-  image: foundationdb/fdb-kubernetes-operator
+---
+image:
+  repository: foundationdb/fdb-kubernetes-operator
   tag: v0.36.0
-  role: fdb-kubernetes-operator-manager-role
-  rolebinding: fdb-kubernetes-operator-manager-rolebinding
-  replicas: 1
-  globalMode:
-    enabled: false
+  pullPolicy: IfNotPresent
+
+initContainers:
+  resources:
+    limits:
+      cpu: 10m
+      memory: 50Mi
+    requests:
+      cpu: 10m
+      memory: 50Mi
+
+  init_6_1:
+    image:
+      repository: foundationdb/foundationdb-kubernetes-sidecar
+      tag: 6.1.13-1
+      pullPolicy: IfNotPresent
+  init_6_2:
+    image:
+      repository: foundationdb/foundationdb-kubernetes-sidecar
+      tag: 6.2.30-1
+      pullPolicy: IfNotPresent
+  init_6_3:
+    image:
+      repository: foundationdb/foundationdb-kubernetes-sidecar
+      tag: 6.3.10-1
```

Comment:
I understand changing these (at least this one that I've tries) are not supported at the moment, but I do hope they will be in the future

Reply:
It should be supported or what error did you see?

Reply:
I failed to realize how the operator is working
I changed it to `tag: 6.3.13-1` and it broke 6.3 versions
I tried with 6.3.13 and 6.3.10 and got a message saying something like "version 6.3.x is not supported"

after figuring it out I've replaced with iteration